src/components/App/App.js

```diff
@@ -0,0 +1,48 @@
+import './App.css';
+
+import {Container, Row, Col} from 'react-bootstrap';
+import DataController from '../DataController/DataController.js'
+import MapController from '../MapController/MapController.js'
+import AreaController from '../AreaController/AreaController.js'
+import React, {useState} from 'react'
+
+function App() {
+  const [currentRegionName, setCurrentRegionName] = useState(null);
+  const [currentRegionBoundary, setCurrentRegionBoundary] = useState(null);
+  
+  function setRegionName(event) {
+      setCurrentRegionName(event);
+  }
+  
+  function setRegionBoundary(event) {
+      setCurrentRegionBoundary(event);
+  }
+
+  return (
+    <div className="App">
+      <header className="App-header">
+        <h1>Salmon Climate Impacts Portal</h1>
+        <Container fluid>
+          <Row>
+            <Col lg={6} md={6}>
+              <MapController
+                currentRegionBoundary={currentRegionBoundary}
+              />
+              <AreaController
+                onChangeRegionName={setRegionName}
+                onChangeRegionBoundary={setRegionBoundary}
+              />
+            </Col>
+            <Col lg={6} md={6}>
+              <DataController
+                currentRegionBoundary={currentRegionBoundary}
+              />
+            </Col> 
+          </Row>
+        </Container>        
+      </header>
+    </div>
+  );
+}
```

Comment:
Container/Row/Column isn't working. All components are stacked vertically. Any suggestions, @rod-glover ? Do I need something in the CSS to make them work?